GOVERNANCE.md

```diff
@@ -25,6 +25,8 @@ The following groups automatically qualify for membership and can request to be
 * OpenJS Foundation CPC Members
 * OpenJS Foundation Project Maintainers
 
+Every team member is asked to document one's involvement in any of the standards organizations that person is regularly involved with.
```

Comment:
I disagree with this framing. I was imagining that we would be documenting who is representing openjs at these meetings.

I participate in a number of standards bodies in my role at Google and don't think it would be appropriate, or even helpful, to document them all here.

Further, I don't think it is a requirement to have to be a member of this working group to have your name included in the list.

I think this paragraph should be removed

Reply:
@MylesBorins makes sense, I will remove it